tools/gce_setup/private_build_and_test.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This script has to be run from the same directory as grpc_docker.sh and after grpc_docker.sh is sourced
+#
+# Sample Usage:
+# ===============================
+# ./private_build_and_test.sh [language] [environment: interop|cloud] [test case]
+#                              [git base directory] [server name in interop environment] 
+# sh private_build_and_test.sh java interop large_unary /usr/local/google/home/donnadionne/grpc-git grpc-docker-server1
+# sh private_build_and_test.sh java cloud large_unary /usr/local/google/home/donnadionne/grpc-git
+# =============================== 
+
+# Arguments
+LANGUAGE=$1
+ENV=$2
+TEST=$3
+GIT=$4
+SERVER=${5:-"grpc-docker-server"}
+
+sudo docker run --name="private_images" -v $4:/var/local/git-clone grpc/$1 /var/local/git-clone/grpc/tools/dockerfile/grpc_$1/build.sh
+
+sudo docker commit -m "private image" -a "donnadionne" private_images grpc/private_images
+
+sudo docker tag -f grpc/private_images 0.0.0.0:5000/grpc/private_images
+
+sudo docker push 0.0.0.0:5000/grpc/private_images
+
+sudo docker rmi -f grpc/private_images
+
+sudo docker rm private_images
+
+gcloud compute --project "stoked-keyword-656" ssh --zone "asia-east1-a" "grpc-docker-testclients1" --command "sudo docker pull 0.0.0.0:5000/grpc/private_images"
```

Comment:
This script should work for everyone who has docker set up and docker-registry running.  I have a document for that.

I did update to use $USER for committing the image 

Craig I am not sure what you mean by default project.  With this script we will be using the same project as our continuous runs but on dedicated private testing vms.   The plan is that if you change only the client or server side, then you can be testing with server or client side that only has official images.  And all VMs are in the same project.